lib/functions/transferpartscreate.js

```diff
@@ -88,7 +88,7 @@ class CreateTransferParts extends AsyncGeneratorFunction {
                         yield transferPart;
                         ++idx;
                     }
-                    logger.info(`${idx} transfer parts created (target uses file protocol)`);
+                    console.log(`${idx} transfer parts created (target uses file protocol)`);
```

Comment:
`logger` doesn't work in Runtime containers ?

Reply:
yep, logger is a wrapper around `debug` which requires us to set an environment variable at the container level